federation/apis/federation/v1beta1/types.go

```diff
@@ -54,6 +54,8 @@ const (
 	ClusterReady ClusterConditionType = "Ready"
 	// ClusterOffline means the cluster is temporarily down or not reachable
 	ClusterOffline ClusterConditionType = "Offline"
+	// FederationNamespaceSystem is the system namespace where we place federation control plane components.
```

Comment:
Create a new `const()` block for this. We generally follow the style of having a block per-related group of things.